modules/nf-core/seqtk/cutn/meta.yml

```
@@ -0,0 +1,41 @@
+name: seqtk_cutn
+description: Generates a BED file containing genomic locations of lengths of N.
+keywords:
+  - cut
+  - fasta
+tools:
+  - seqtk:
+      description: Seqtk is a fast and lightweight tool for processing sequences in the FASTA or FASTQ format. Seqtk mergepe command merges pair-end reads into one interleaved file.
+      homepage: https://github.com/lh3/seqtk
+      documentation: https://docs.csc.fi/apps/seqtk/
+      tool_dev_url: https://github.com/lh3/seqtk
+      licence: ["MIT"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
+      type: file
+      description: A single fasta file to be split.
+      pattern: "*.{fasta}"
+
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - reads:
```

Change `reads` to `fasta` to match the `main.nf`
```suggestion
  - bed:
```